tests/test_views.py

```diff
@@ -302,7 +302,19 @@ def test_env_per_page(bukudb, app, client, total, per_page, pages, last_page):
 @pytest.mark.parametrize('favicons', [False, True, None])
 @pytest.mark.parametrize('mode', ['full', 'netloc', 'netloc-tag', None])
 def test_env_entry_render_params(bukudb, app, client, mode, favicons, new_tab):
-    url, netloc, title, desc, tags = 'http://example.com', 'example.com', 'Sample site', 'Foo bar baz', ',bar,baz,foo,'
+    _test_env_entry_render_params(bukudb, app, client, mode, favicons, new_tab, 'http://example.com', 'example.com', 'Sample site')
+
+@pytest.mark.parametrize('url, netloc, title', [
+    ('http://example.com', 'example.com', ''),
+    ('javascript:void(0)', '', 'Sample site'),
+    ('javascript:void(0)', '', ''),
+])
+@pytest.mark.parametrize('mode', ['full', 'netloc', 'netloc-tag'])
+def test_env_entry_render_params_blanks(bukudb, app, client, mode, url, netloc, title):
+    _test_env_entry_render_params(bukudb, app, client, mode, True, True, url, netloc, title)
```

Comment:
Added the test for rendering of blank title/netloc first for convenience (based on behaviour implemented in current `master`).